tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -727,6 +773,12 @@ def _get_alert_activations(project_name, alert_name=None):
             project=project_name, name=alert_name
         )
 
+    @staticmethod
+    def _get_alert_config_activations(alert, from_last_update=False, paginated=False):
+        if paginated:
+            return alert.paginated_list_activations(from_last_update=from_last_update)
+        return alert.list_activations(from_last_update=from_last_update)
+
```

Comment:
We have `self._get_alert_activations` already which allows passing alert name, please either use it or pure methods

Reply:
but i dont want to pass the alert name, since this should be called from the alert config object itself, and the existing method go through the project,  so it may be better to separate between them.

Reply:
@Yacouby then won't it be better to use alert.list_activations() and alert.paginated_list_activations?